solution_runner/commands/commands_utils.py

```diff
@@ -0,0 +1,89 @@
+import urllib.parse
+from datetime import datetime
+from enum import Enum
+from pathlib import Path
+from typing import Any
+
+import click
+import requests
+import yaml
+
+from . import consts
+
+
+class PathType(Enum):
+    FILE = 1
+    DIRECTORY = 2
+
+
+def get_setting(key: str) -> Any:
+    """
+    :param key: key to retrieve its value
+    :return: value of `key` which is stored in the configuration file
+    """
+    configuration_file = consts.APP_DATA_DIRECTORY / consts.CONFIGURATION_FILE_NAME
+    try:
+        configuration = yaml.safe_load(configuration_file.read_text())
+    except FileNotFoundError:
+        click.secho(
+            "configuration file doesn't exist. Run config command first", fg="red"
+        )
+        raise
+
+    return configuration[key]
+
+
+def check_path_exists(path: Path, command: click.Command, path_type: str = None):
+    """
+    Abort if path doesn't exist.
+    :param path: path to check
+    :param command: command to be passed to`click.Context` if aborting is needed
+    :param path_type: optional "dir" or "file". If not passed, only check for existence. If passed, check for type.
```

Comment:
It is better to have separate functions for each case (`exists`/`is_dir`/`is_file`) instead of a string parameter: `check_dir_exists(...)` is nicer than `check_path_exists(..., "dir")`.
See https://wiki.c2.com/?StringlyTyped.

Reply:
Apperently I am not using it so for now I will just delete it (afdec7d2).

Reply:
Apparently, I am not using it, so I deleted it (afdec7d2).